source/_docs/migrate.md

```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
+```
+
+<div class="alert alert-danger" role="alert">
+<h4 class="info">Warning</h4>
+<p markdown="1">We recommend using a random hash for the archive filename for security. Archive dumps contain sensitive information, so they should not be exposed using guessable filename patterns (like `BACKUP` or recent dates).</p>
+</div>
+
+Click **Continue Migration** and follow all remaining instructions within the guided migration process.
+
+**Be sure to delete the Drupal archive file (`<RANDOM_HASH>.tgz`) from your source site after cloning the site, using SFTP to access your site's files.**
+
+If your archive is larger than 256MB the Terminus operation to generate the archive will fail, resulting in the following error:
+
+```bash
+Connection to appserver.<ENV>.<Site UUID>.drush.in closed by remote host.
+```
+
+If your database and code compressed are less than 256MB you can exclude the files directory from export using the `--tar-options="--exclude=code/sites/default/files"` flag. Then you can download a backup of the files from the existing site and import the archive to the new site within **<span class="glyphicons glyphicons-server"></span> Database / Files** > **Import**.
 
-#### How do I migrate a local site to Pantheon?
-You can import a WordPress or Drupal site archive via URL (within file size limits) using [Terminus](/docs/terminus):
+### How do I migrate a local site to Pantheon?
+You can import a WordPress site archive via URL (within file size limits) using [Terminus](/docs/terminus):
 
 ```bash
 terminus site:import <site> <url>
 ```
 
 You'll need to [manually migrate](/docs/migrate-manual) site archives that exceed 500MB.
 
-#### How do I migrate a Drupal 6 site to Pantheon?
-Anyone wishing to migrate a Drupal 6 site to Pantheon can work with one of our Long Term Support (LTS) partners: [Tag1 Consulting](https://tag1consulting.com/) or [myDropWizard](https://www.mydropwizard.com/drupal-6-lts). Both of these partners are experienced in supporting sites on the Pantheon platform and specialize in maintaining security and site functionality for Drupal 6 sites. Should you need to keep your site running on D6, you will be in excellent hands working with them.
+### How long does the WordPress migration process take?
+Most migrations are completed within two hours. The migration time depends solely on the size of your site, so be aware that it may take more or less time than estimated. We will send you an email once your migration is complete. If there are any issues with the migration, we will notify you by email.
+
+### Why doesn't my site archive import correctly from Dropbox?
+Change the end of Dropbox URLs from `dl=0` to `dl=1` so we can import your site archive properly. See the Dropbox documentation: [How do I force a file or folder to download or render on dropbox.com?](https://www.dropbox.com/en/help/201)
+
+### How do I use the Pantheon Migrations plugin with a custom WordPress upstream?
```

Comment:
I think this section could be clarified. It says that the process is the same, but by clicking on **Create New Site** the user won't get the same options. If I understand correctly, they need to do something else to manually initiate the migration, yes? Does this section need a reference to the Terminus command used in the next section?